packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -162,9 +180,36 @@ dotenv.config({ path: "../graph-explorer/.env" });
     let response;
     let data;
     try {
-      response = await retryFetch(`${req.headers["graph-db-connection-url"]}/?gremlin=` +
-      encodeURIComponent(req.query.gremlin), undefined, undefined, req, "gremlin").then((res) => res)
+      response = await retryFetch(
+        `${req.headers["graph-db-connection-url"]}/?gremlin=` +
+          encodeURIComponent(req.query.gremlin),
+        undefined,
+        undefined,
+        req,
+        "gremlin"
+      ).then((res) => res);
+
+      data = await response.json();
+      res.send(data);
+    } catch (error) {
+      next(error);
+      console.log(error);
+    }
+  });
+
+  app.get("/pg/statistics/summary", async (req, res, next) => {
+    let response;
+    let data;
+    try {
+      response = await retryFetch(
+        `${req.headers["graph-db-connection-url"]}/pg/statistics/summary`,
+        undefined,
+        undefined,
+        req,
+        "gremlin"
+      ).then((res) => res);
 
+      console.log(response);
```

Comment:
The response from the graph summary endpoint can be pretty big for certain types of graphs.
Could there be any issues with logging the entire response to console in those cases?

Same question for L233 (log statement in rdf summary endpoint)

Reply:
@nestoralvarezd to comment

Reply:
I can remove this kind of large logging